presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -45,6 +46,11 @@ public HiveSessionProperties(HiveClientConfig config)
     {
         sessionProperties = ImmutableList.of(
                 booleanSessionProperty(
+                        BUCKET_EXECUTION_ENABLED,
+                        "Assign one bucket to each worker",
```

Comment:
Actually, let's use the same description as the config: "Use bucketing to speed up execution"